apps/src/applab/Exporter.js

```diff
@@ -466,19 +464,22 @@ export default {
     const appOptionsJs = getAppOptionsFile();
     const { css, outerHTML } = transformLevelHtml(levelHtml);
     const fontAwesomeCSS = exportFontAwesomeCssEjs({fontPath: fontAwesomeWOFFPath});
-    const exportBaseHref = `https://studio.code.org/projects/applab/${project.getCurrentId()}/`;
-    const exportConfigPath = getExportConfigPath(exportBaseHref);
+    const exportConfigPath = getExportConfigPath();
+    const { origin } = window.location;
```

Comment:
🤦thanks for obtaining this in a way that is much simpler than what I was recommending.